### PR TITLE
test/test_helpers.js: use `path.extname` to get the extension

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: test/test_helpers.js [hunk 1]
```diff
@@ -22,14 +23,7 @@ let server = {};
...
 function getExtension(str) {
-    // use two enclosing parts; one for the dot (.)
-    // and one for the extension itself.
-    // So, the result we want is the third Array element,
-    // since the first one is the whole match, the second one
-    // returns the first captured match, etc.
-    const match = str.match(/(\.)([a-zA-Z0-9]+)$/);
-
-    return match && match[2];
+    return path.extname(str).slice(1);
```
Comment (XhmikosR) [inline test/test_helpers.js]: Note that str is a URL

Comment (XhmikosR) [reply test/test_helpers.js]: I'm still hesitant about this because str is a URL...

Comment (jmervine) [reply test/test_helpers.js]: What will it return if there's no extension? Generally I think it's okay that it's a URL if (a) it returns `nil` or `undefined` when there's no extension or (b) we can be certain that all `str` values passed contain extensions.

Comment (XhmikosR) [reply test/test_helpers.js]: An empty string is returned if there's no extension but then our tests will fail later anyway.

I will rethink about this.

**Pull request merged**